proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -33,6 +33,10 @@ service ResolveService {
   rpc GetModulePins(GetModulePinsRequest) returns (GetModulePinsResponse) {
     option idempotency_level = NO_SIDE_EFFECTS;
   }
+
+  rpc ResolveReference(ResolveReferenceRequest) returns (ResolveReferenceResponse) {
```

Comment:
Not sure if this should have a more descriptive name relating to registries? maybe something like `ResolveRegistryVersion`?

Reply:
Regardless of the name, some docs on this as to what it is would be good